addons/MagicaVoxelImporter/VoxelMesh.gd

```diff
@@ -25,8 +28,15 @@ func _set_neck_height(value):
 		for mat in _get_mats():
 			mat.set_shader_param('neck_height', value)
 
-func _set_render_head(value):
-	render_head = value
-	if mesh:
-		for mat in _get_mats():
-			mat.set_shader_param("render_head",value)
+func set_render_head(value):
+	if is_inside_tree():
+		render_head = value
+		var vox_char_node := $"." as MeshInstance
```

Comment:
`$"."` is just `self` so you can just remove the `vox_char_node` variable and access these methods with the implied `self`

`var mat = vox_char_node.get_surface_material(0)` -> `var mat = get_surface_material(0)` etc.

Reply:
Done